src/workers/utils/name-verification.ts

```diff
@@ -1,13 +1,21 @@
+import path from 'path';
+
 const isWindowsRootDirectory = /[a-zA-Z]:[\\/]/;
 const containsNullCharacter = /\0/g;
 
 const validations = [
   (name: string) => name.includes('../'),
```

Comment:
One little thing I can think of here, not urgent though is

`validations: ((name: string) => boolean)[] = [`

So you don't have to type all name as strings below @JoanVicens